packages/utils/src/calculateTheme.ts

```diff
@@ -1,4 +1,4 @@
-import { createTheme, ThemeOptions } from "@material-ui/core/styles";
+import { createTheme, DeprecatedThemeOptions, adaptV4Theme } from "@mui/material/styles";
```

Comment:
I am ok to punt this to a different PR. Would you mind create an issue to update the theme calculation to use v5 interfaces?

Reply:
https://github.com/stateful/vscode-marquee/issues/49